conda/install.py

```diff
@@ -1048,17 +1053,7 @@ def link(prefix, dist, linktype=LINK_HARD, index=None, shortcuts=False):
                 os.makedirs(dst_dir)
             if os.path.exists(dst):
                 log.warn("file already exists: %r" % dst)
-                try:
-                    os.unlink(dst)
-                except OSError:
-                    log.error('failed to unlink: %r' % dst)
-                    if on_win:
-                        try:
-                            move_path_to_trash(dst)
-                        except ImportError:
-                            # This shouldn't be an issue in the installer anyway
-                            pass
-
+                rm_rf(dst)
```

Comment:
Is it? We should be able to fix this. I think if we do a `conda install -f` integration test it will hit this.